src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,53 @@
+.container {
+  padding: 0 20px;
+
+  @include tablet {
+    padding: 0 27px;
+  }
+
+  @include desktop {
+    padding: 0 50px;
+    margin: auto;
+  }
+
+  @include big-desktop {
+    padding: 0 130px;
+    margin: auto;
+  }
+}
+
+html {
```

Comment:
The task description mentions 'Sharp Sans No2 Medium' and 'Avenir' as fonts. However, 'Poppins' is used in the code. Please correct the font family to match the task specification.